_episodes/08-testing.md

```diff
@@ -130,7 +130,7 @@ You should see an output similar to the following.
 
 ```{code-block} output
 ============================= test session starts ==============================
-platform darwin -- Python 3.6.8, pytest-3.6.4, py-1.5.4, pluggy-0.6.0
+platform darwin -- Python 3.10.12, pytest-7.4.2, pluggy-1.3.0
```

Comment:
Should these be python 3.11?

Reply:
Yes you are correct, it  is suppose to be 3.11.6. I will fix that.